src/Symfony/Component/HttpFoundation/Response.php

```diff
@@ -169,7 +169,7 @@ class Response
         422 => 'Unprocessable Entity',                                        // RFC4918
         423 => 'Locked',                                                      // RFC4918
         424 => 'Failed Dependency',                                           // RFC4918
-        425 => 'Reserved for WebDAV advanced collections expired proposal',   // RFC2817
+        425 => 'Too Early',                                                   // RFC-ietf-httpbis-replay-04
```

Comment:
I guess we also need to add a new constant above:

```php
const HTTP_TOO_EARLY = 425;
```

and leave the old one to maintain BC:

```php
const HTTP_RESERVED_FOR_WEBDAV_ADVANCED_COLLECTIONS_EXPIRED_PROPOSAL = 425;
```

Reply:
Done, I also deprecated the old const.